test/controllers/ChargeSummaryControllerSpec.scala

```diff
@@ -390,18 +390,19 @@ class ChargeSummaryControllerSpec extends ChargeSummaryControllerHelper {
               JsoupParse(result).toHtmlDocument.select("h1").text() shouldBe errorHeading
             }
 
-            if (mtdUserRole == MTDIndividual) {
-              "no related tax year financial details found" in new Setup(testFinancialDetailsModelWithPayeSACodingOut()) {
-                disable(ChargeHistory)
-                setupMockSuccess(mtdUserRole)
-                mockBothIncomeSources()
-
-                val result: Future[Result] = testController.show(2020, "CODINGOUT01")(fakeRequest)
-
-                status(result) shouldBe Status.INTERNAL_SERVER_ERROR
-                JsoupParse(result).toHtmlDocument.select("h1").text() shouldBe errorHeading
-              }
-            }
+            //TODO: Check validity of this test
```

Comment:
do we really need this commented code?

Reply:
Oh yep sorry, still need to investigate this test. Doing now

Reply:
Not needed as we don't group by tax year anymore